html/index.html

```diff
@@ -3,9 +3,213 @@
 <head>
   <meta charset="UTF-8">
   <title>Document</title>
+  <link href='css/less/styles.css' rel='stylesheet'>
+  <link href="https://fonts.googleapis.com/css?family=Raleway:700|Montserrat&display=swap" rel="stylesheet">
 </head>
 <body>
-  
-    <script src="http://localhost:35729/livereload.js"></script>
+  <nav>
```

Comment:
we should wrap the `<nav>` with `<header>`

Reply:
currently this; should change?
<header>
  <p>Name</p>
  <nav>
    <a>link</a>
    <a>...</a>
  </nav>
</header>